Formula/doggo.rb

```diff
@@ -0,0 +1,29 @@
+class Doggo < Formula
+  desc "Command-line DNS Client for Humans"
+  homepage "https://doggo.mrkaran.dev/"
+  url "https://github.com/mr-karan/doggo.git",
+      tag:      "v0.5.3",
+      revision: "ea7cb3c6cd0467a7db783fbd30aa8599604a76d8"
+  license "GPL-3.0-or-later"
```

Comment:
```suggestion
  url "https://github.com/mr-karan/doggo/archive/refs/tags/v0.5.3.tar.gz"
  sha256 "4f8333662fd73c411bf9fb7ce8254d526a777b6491c1d5a49fff0f09b00754c1"
  license "GPL-3.0"
```

Reply:
need to confirm the license as GPL-3.0 is deprecated.

Reply:
Tests now fail because of deprecated license spec.